worker/pkg/workflows/datasync/activities/activities_test.go

```diff
@@ -206,6 +210,114 @@ output:
 	require.NoError(t, err)
 }
 
+type ExpectedJsValue struct {
+	Name string
+}
+
+func Test_Sync_Run_Success_Javascript(t *testing.T) {
+	testSuite := &testsuite.WorkflowTestSuite{}
+	env := testSuite.NewTestActivityEnvironment()
+
+	activities := &Activities{}
+	env.RegisterActivity(activities)
+
+	// benthos writes to stdout, need to capture that to evaluate the returned value, so we create a pipe to capture what is written to stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
```

Comment:
idk about this. Looks like you are using the global Stdout to be this os.Pipe for a single test.
How does this work for the rest of the tests?

Reply:
i was trying to come up with a way to actually test the transformed value output and since benthos writes to stdout for `codec:lines` i'm piping that and then getting the resulting value. I wasn't able to find anywhere else where we were actually processing the benthos config and checking the output vs. just the formatting and structure of the benthos config. Open to other ideas.

Reply:
yea you're right we don't have any tests today that test the benthos output at an integration level.
A better approach here might be to have benthos output to a file in the temp directory that you can then read from afterwards to verify the output. That way there is no need to hijack the stdout.

Reply:
updated